protocol/server_parameters.go

```diff
@@ -2,6 +2,10 @@ package protocol
 
 import "time"
 
+// NonForwardSecurePacketSizeReduction is the number of bytes a non forward-secure packet has to be smaller than a forward-secure packet
+// This makes sure that those packets can always be retransmitted without splitting the contained StreamFrames
+const NonForwardSecurePacketSizeReduction = 50
```

Comment:
I'll create an issue to refactor the retransmission logic of handshake packets, such that we can get rid of this `NonForwardSecurePacketSizeReducton`. It works for now, but it's not a clean solution anyway, and it won't get a lot cleaner by calculation the exact value here.